src/event_pusher/mod_event_pusher_http.erl

```diff
@@ -59,8 +59,12 @@ make_req(Host, Sender, Receiver, Message) ->
     Path = fix_path(list_to_binary(gen_mod:get_module_opt(Host, ?MODULE, path, ?DEFAULT_PATH))),
     PoolName = gen_mod:get_module_opt(Host, ?MODULE, pool_name, ?DEFAULT_POOL_NAME),
     Pool = mongoose_http_client:get_pool(PoolName),
-    Query = <<"author=", Sender/binary, "&server=", Host/binary,
-              "&receiver=", Receiver/binary, "&message=", Message/binary>>,
+    EncodedHost = cow_qs:urlencode(Host),
+    EncodedSender = cow_qs:urlencode(Sender),
+    EncodedReceiver = cow_qs:urlencode(Receiver),
+    EncodedMessage = cow_qs:urlencode(Message),
+    Query = <<"author=", EncodedSender/binary, "&server=", EncodedHost/binary,
```

Comment:
usage of cow_qs:qs/1 can be considered slower, but more readable

Reply:
Definitely. I think the cost of executing `cow_qs:qs/1` is very small, compared to the cost of HTTP call.